pegasus/sites.v3/code.org/public/about/index.erb.md.erb

```diff
@@ -124,6 +124,9 @@ Code.org IRS form 990 for [2014](/files/irs-form.pdf), [2015](/files/irs-form-20
 
 <hr/>
 
-<a href="http://www.guidestar.org/organizations/46-0858543/code-org.aspx" target="_blank">
-    <img src="https://widgets.guidestar.org/gximage2?o=9218725&l=v3" />
-</a> [![image](/images/fit-150/student_privacy_pledge.png)](http://studentprivacypledge.org/)
+<a href="https://www.guidestar.org/profile/46-0858543" target="_blank" rel="noopener noreferrer">
+  <object data="https://widgets.guidestar.org/gximage2?o=9218725&l=v4" type="image/png" style="vertical-align: middle; margin-right: 40px">
+    <img src="/images/guidestar-logo.png" />
+  </object>
```

Comment:
This is an interesting technique.  Is there any visual weirdness in any browsers where we see one image load and then the other overrides it?  Or do we always just see one image load?